unlock-app/src/middlewares/postOfficeMiddleware.ts

```diff
@@ -18,20 +18,34 @@ const postOfficeMiddleware = (window: IframePostOfficeWindow, config: any) => {
   // No account yet, will be set after login
   postOfficeService.setAccount(null)
 
+  // Locks on the paywall, keys are lower-cased lock addresses
+  // no need for a reducer here because this state is entirely local to the
+  // postOffice
+  let locksOnPaywall: { [key: string]: any } = {}
```

Comment:
Let's use the `Locks` type here in place of `any`, it will give you auto-completion elsewhere

Reply:
Wasn't sure if this was strictly the same as the type we defined in `unlockTypes` since postOfficeService passed them through as `any`. I'll update in both spots if that's ok.

Reply:
technically, it can be tightened up quite a bit, I have that branch sitting open that does this in a flexible way (limits types based on the kind of message posted), but if it breaks things to not use `any` let's keep it for now, and tighten later.

Reply:
I'm not 100% that it will break anything, but let's move forward on this one and then clean up